aptos-move/aptos-aggregator/src/lib.rs

```diff
@@ -5,7 +5,6 @@ pub mod aggregator_extension;
 pub mod delta_change_set;
 mod module;
 pub mod resolver;
-pub mod transaction;
```

Comment:
Nice :) I feel like we will remove aggregator-crate soon (in favour of vm-types?)